dts/arm/st/stm32f407.dtsi

```diff
@@ -4,10 +4,15 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <st/stm32f407-pinctrl.dtsi>
 #include <st/stm32f411.dtsi>
 
 / {
 	soc {
+		pinctrl: pin-controller {
+			reg = <0x40020000 0x2800>;
+		};
```

Comment:
Actually this is only the delta vs initial pincntrl node located in stm32f4.dtsi